src/main/java/mate/academy/dao/impl/MovieDaoImpl.java

```diff
@@ -45,6 +45,13 @@ public Optional<Movie> get(Long id) {
 
     @Override
     public List<Movie> getAll() {
-        return null;
+        Session session = HibernateUtil.getSessionFactory().openSession();
+        try {
+            return session.createQuery("From Movie", Movie.class).getResultList();
+        } catch (Exception e) {
+            throw new RuntimeException("Can't get list of comments", e);
```

Comment:
The exception message in the `getAll` method refers to 'comments' instead of 'movies'. Consider updating the message to 'Can't get list of movies' for clarity and consistency.